tests/unit/date-mixin.spec.ts

```diff
@@ -20,8 +20,7 @@ describe('Date Mixin', () => {
     await Vue.nextTick()
   })
 
-  // FUTURE: fix so this works in GH CI action
```

Comment:
This works now that the Date Mixin uses the lowest common denominator for local ("en-US").